### ISSUE TITLE
TypeError when converting po file to csv

### ISSUE THREAD

**Issue by kngai** [marek-saji/po-csv#4]
Hi @marek-saji,

I really like what your plugin offers, but I get an error on my first attempt. I'm not sure what to make of it.

Here is my trace (I'm using Windows Powershell):

```
PS C:\path\to\my_project\node_modules\po-csv> node .\index.js fr.po > untranslated.csv
C:\path\to\my_project\node_modules\fast-csv\lib\formatter\formatter.js:169
        vals[i] = item[i][1];
                         ^

TypeError: Cannot read property '1' of undefined
    at transformHashArrayData (C:\path\to\my_project\node_modules\fast-csv\lib\formatter\formatter.js:169:26)
    at transformItem (C:\path\to\my_project\node_modules\fast-csv\lib\formatter\formatter.js:180:19)
    at C:\path\to\my_project\node_modules\fast-csv\lib\formatter\formatter_stream.js:62:42
    at CsvTransformStream.__transform (C:\path\to\my_project\node_modules\fast-csv\lib\formatter\formatter_stream.js:79:17)
    at CsvTransformStream._transform (C:\path\to\my_project\node_modules\fast-csv\lib\formatter\formatter_stream.js:57:14)
    at CsvTransformStream.Transform._read (_stream_transform.js:186:10)
    at CsvTransformStream.Transform._write (_stream_transform.js:174:12)
    at doWrite (_stream_writable.js:397:12)
    at writeOrBuffer (_stream_writable.js:383:5)
    at CsvTransformStream.Writable.write (_stream_writable.js:290:11)
```

Any ideas?

**Comment by marek-saji** [comment]
Can you provide `po` file that causes the problem?

**Comment by marek-saji** [comment]
and version of node you are using

**Comment by kngai** [comment]
node: `v8.11.2`
npm: `5.6.0`

Sample po file here: https://gist.github.com/kngai/db330f6c2530cf4ba3a7c4f3fcc998bf
- contains a mix of translated and untranslated parts

**Comment by marek-saji** [comment]
Just published version 1.0.2 that should fix the problem, thanks for reporting. 🙂

**Comment by kngai** [comment]
Thanks for the quick turn around!

I can confirm it's working on my end now. :)
